test/bigtest/tests/job-profiles/jobProfileDetails-test.js

```diff
@@ -27,6 +33,46 @@ describe('Job profile details: default job profile', () => {
     expect(jobProfileDetails.isPresent).to.be.true;
   });
 
+  describe('navigating to deleting confirmation modal', () => {
+    const callout = new CalloutInteractor();
+
+    beforeEach(async () => {
+      await jobProfileDetails.actionMenu.click();
+      await jobProfileDetails.actionMenu.deleteProfileButton.click();
+      await wait();
+    });
+
+    describe('deleting job profile successfully', () => {
+      beforeEach(async function () {
+        await jobProfileDetails.deletingConfirmationModal.confirmButton.click();
+      });
+
+      it('should navigate to job profiles settings page', function () {
+        expect(this.location.pathname.endsWith('/data-export/job-profiles')).to.be.true;
+      });
+
+      it('should display error callout', function () {
```

Comment:
Should in case of `deleting job profile successfully` the error callout be not displayed?